src/motor_node.cc

```diff
@@ -189,9 +230,19 @@ void initMcbParameters(std::unique_ptr<MotorHardware> &robot )
         mcbStatusPeriodSec.sleep();
     }
 
+    // Certain 4WD robots rely on wheels to skid to reach final positions.
+    // For such robots when loaded down the wheels can get in a state where they cannot skid.
+    // This leads to motor overheating.  This code below sacrifices accurate odometry which
+    // is not achievable in such robots anyway to relieve high wattage drive power when zero velocity.
+    g_wheel_slip_nulling = 0;
+    if ((robot->firmware_version >= MIN_FW_WHEEL_NULL_ERROR) && (g_node_params.drive_type == "4wd")) {
+        g_wheel_slip_nulling = 1;
+        ROS_INFO("Wheel slip nulling will be enabled for this 4wd system when velocity remains at zero.");
+    }
+
     // Tell the MCB board what the I2C port on it is set to (mcb cannot read it's own switchs!)
```

Comment:
Wait, the MCB doesn't know it's i2c address? What? Do we... use i2c on the MCB?

Reply:
The MCB was designed with a puny little micro so it is way way out of pins.  Thus this is UGLY as all get-out but sadly is required.   Host reads the option port on I2C and then tells a register in the MCB what it's board rev is.  Don't get me started.   On all Gen6 boards I have insisted with a heavy hand that ALL boards with processors will read and know their own hardware revision and THEN report THAT to the host.

Reply:
Ah I see, well I sure hope we go for some spare flash memory and more pins next time 😄